clients/fides-js/src/fides.ts

```diff
@@ -224,20 +224,12 @@ const init = async ({
     );
 
     if (shouldInitOverlay) {
-      // Check if there are any notices within the experience that do not have a user preference
-      const noticesWithNoUserPreferenceExist: boolean = Boolean(
-        effectiveExperience?.privacy_notices?.some(
-          (notice) => notice.current_preference == null
-        )
-      );
```

Comment:
This check also happens inside the `experienceIsValid` function, right...? That feels duplicative and probably shouldn't exist in the experienceIsValid function at all anymore?